package.json

```diff
@@ -17,17 +17,19 @@
   "author": "Ian Burns <iwburns8@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "@types/chai": "^4.0.4",
-    "@types/mocha": "^2.2.43",
+    "@types/chai": "^4.1.2",
+    "@types/es6-shim": "^0.31.35",
+    "@types/mocha": "^2.2.48",
     "chai": "^4.1.2",
     "coveralls": "^3.0.0",
-    "mocha": "^4.0.1",
+    "es6-shims": "^2.0.1",
```

Comment:
I'm probably just missing it somewhere, but where is this being used?

Reply:
I had been using promises in the tests, I changed that though so ill pull it back out